### PR TITLE
Update Debug Logging Documentation

### PR DESCRIPTION
**Pull request by srcrist**
Expanded and revised the documentation on debug logging to include the case where an installation has the optional 'logging.json' file present. Also generally revised and clarified language, added a link to the configuration directory documentation, and added instructions for disabling debug logging.

### CODE REVIEW

File: docs/general/administration/troubleshooting.md [hunk 1]
```diff
@@ -28,8 +28,10 @@ This would indicate either an incorrect address or an issue somewhere else on th
 
 ## Debug Logging
 
-To enable debug (much more verbose) logging, it is currently required to manually edit config files since no options exist yet on the frontend.
-Go to the Jellyfin configuration directory, find the `logging.default.json` file, and change the minimum level to debug as seen below.
+To enable much more verbose debug logging, it is currently required to manually edit configuration files since no option yet exists to enable debug functionality within the frontend UI.
+Go to the [Jellyfin configuration directory](/docs/general/administration/configuration#configuration-directory) and find the `logging.json` file if it exists, or the `logging.default.json` file if it does not. Debug logging is then enabled by changing the minimum logging level to debug as in the examples below.
+
+The `logging.default.json` file should be modified as below.
```
Comment (thornbill) [inline docs/general/administration/troubleshooting.md:34]: The default file should be copied to `logging.json` and modifications made there.

Comment (srcrist) [reply docs/general/administration/troubleshooting.md]: Conversation resolved on Matrix. An overhaul will be written to remove this methodology.

File: docs/general/administration/troubleshooting.md [hunk 2]
```diff
@@ -28,21 +28,35 @@ This would indicate either an incorrect address or an issue somewhere else on th
...
+
+Once the need for verbose logging has passed, debug logging can be disabled by changing the `"MinimumLevel"` key in `logging.json` to `"Information"`, as in the example below, in order to restore the default logging level. It is not necessary to delete the `logging.json` configuration after debugging is complete.
+
+```json
+{
+  "Serilog": {
+    "MinimumLevel": "Information"
+    }
+}
+```
```
Comment (nielsvanvelzen) [inline docs/general/administration/troubleshooting.md:59]: Indenting is a bit weird, I think that's also the case with the previous codeblock;

```suggestion
```json
{
  "Serilog": {
    "MinimumLevel": "Information"
  }
}
```
```

Comment (srcrist) [reply docs/general/administration/troubleshooting.md]: Yeah, I think I just copied the code block. Yours looks better.

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**